megamek/src/megamek/common/Entity.java

```diff
@@ -15903,4 +15903,11 @@ public Base64Image getBase64Icon() {
     public boolean countForStrengthSum() {
         return !isDestroyed() && !isTrapped() && !isPartOfFighterSquadron();
     }
+
+    /** @return True if the unit should use Edge based on the current options and assigned Edge points */
+    public boolean shouldUseEdge(String option) {
+          return (game.getOptions().booleanOption(OptionsConstants.EDGE)
+              && getCrew().hasEdgeRemaining()
```

Comment:
For safety, will need `&& getCrew() != null` immediately above this line.

Reply:
Good call, done in 16d2e9c